src/components/ArtifactDataImage.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="artifact-data-image">
+    <img loading="eager" :src="artifact.data">
```

Comment:
nit: eager is the default
```suggestion
    <img :src="artifact.data">
```